tests/common/dualtor/dual_tor_mock.py

```diff
@@ -350,6 +350,8 @@ def apply_peer_switch_table_to_dut(cleanup_mocked_configs, rand_selected_dut, mo
         logger.info("Restarting swss service to regenerate config.bcm")
         dut.shell('systemctl reset-failed swss; systemctl restart swss')
         wait_critical_processes(dut)
+        logger.info("Wait for 100 seconds for complete update of configs after swss restart")
+        time.sleep(100)
```

Comment:
could you please elaborate why do we need wait for 100s?

Reply:
Can we improve this with a `wait_until` that has a check to see if the tunnel intf is up and ready?

Reply:
Reason for 100s wait : 
[This PR](https://github.com/sonic-net/sonic-buildimage/pull/15741) is skipping the wait for tunnel for mock dualtor - 
the reason for this is described in the PR, but basically since we are skipping wait_for_tunnel on mocked dualtor, we are skipping the 90s wait time that was earlier meant for tunneling - however previously, during this 90s wait time application and updating of configs after swss restart was completed, and CRM was also updated. Therefore, when we skip this, configs are not applied, and CRM is not updated, and therefore this 100s wait time after swss restart compensates for the same.